pkg/store/copr/coprocessor.go

```diff
@@ -1825,18 +1832,32 @@ func (worker *copIteratorWorker) collectCopRuntimeStats(copStats *CopRuntimeStat
 	}
 	copStats.ScanDetail = sd
 	copStats.TimeDetail = td
+
+	if worker.req.RunawayChecker != nil {
+		var ruDetail *util.RUDetails
+		if ruDetailRaw := bo.GetCtx().Value(util.RUDetailsCtxKey); ruDetailRaw != nil {
+			ruDetail = ruDetailRaw.(*util.RUDetails)
+		}
+		if err := worker.req.RunawayChecker.CheckThresholds(ruDetail, sd.ProcessedKeys, nil); err != nil {
```

Comment:
`sd.ProcessedKeys` is the processed keys of the current copr request, but I suppose we should compare the total processed_keys of all copr tasks.